webpack.config.js

```diff
@@ -45,14 +47,8 @@ const plugins = [
     filename: 'index_admin.html',
     chunks: ['admin']
   }),
-  new VersionFile({
```

Comment:
@maxjvh what was this for? I'm not sure why it was removed, might have been incompatibility stuff. I can add it back if it's something that's needed.